manifests/02-rbac-role.yaml

```diff
@@ -20,6 +20,7 @@ rules:
 - apiGroups:
   - ""
   resources:
+  - pods
```

Comment:
So we don't directly manipulate `pods`, why do we need to add this?

Reply:
Explained in comment above.